macOS_Catalina.md

```diff
@@ -1,39 +1,37 @@
 # Installation notes for macOS Catalina (v10.15)
 
-_This document specifically refers to upgrades from previous versions of macOS to Catalina (10.15). It should be removed from the source repository when Catalina ceases to be the latest macOS version or updated to deal with challenges involved in upgrades to the next version of macOS._
+_This document specifically refers to upgrades from previous versions of macOS to Catalina (10.15). It should be removed from the source repository when Catalina ceases to be the latest macOS version or when Catalina has been fixed to solve these problems._
```

Comment:
I'm a little uncomfortable with "has been fixed", can we say for sure this is a macOS _bug_ that needs to be fixed and not just some side effect of the way they want people to interact with their systems? I still haven't upgraded the mac I have to Catalina so haven't gone through any of this pain so my perspective is more limited. If you (and @cclauss) are comfortable asserting that this is something deserving of a "fix" and not just something they could address to help deal with the challenges of dealing with a diverse ecosystem then that's fine. Otherwise, this should be softened a little I think.

Reply:
Fair point. Substantially softened.